staging/src/k8s.io/apiserver/pkg/storage/cacher/testing/lister_watcher_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package cacher
+package testing
```

Comment:
Why do you need to touch this file? I think it can stay where it was which is more natural place for it.

Reply:
It is, however it will create a dependency cycle: cacher -> etcdsetup -> cacher